### PR TITLE
MeshPhysicalMaterial: Add `dispersion`.

### LINKED ISSUE #28001: Support for KHR_materials_dispersion
### Description

I'm interested in trying a dispersion material in a glTF model and would like to be able to see the effect in three.js

### Solution

Implement support for [KHR_materials_dispersion](https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_materials_dispersion) 

### Alternatives

None, to my knowledge

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Mugen87**
Fixed #28001.

**Description**

This PR adds dispersion support to `MeshPhysicalMaterial` based on [KHR_materials_dispersion](https://github.com/KhronosGroup/glTF/tree/main/extensions/2.0/Khronos/KHR_materials_dispersion).

It enhances `GLTFLoader`, the editor and adds a new example `webgl_loader_gltf_dispersion`. I have temporarily updated the build files so it can be tested here:

https://rawcdn.githack.com/Mugen87/three.js/55c6ae017d7953a4040d8fdbe00fbb7f9cd31c49/examples/webgl_loader_gltf_dispersion.html

For comparison this is the same model with the Khronos glTF Sample Viewer: https://github.khronos.org/glTF-Sample-Viewer-Release/?model=https://raw.GithubUserContent.com/KhronosGroup/glTF-Sample-Assets/main/./Models/DispersionTest/glTF-Binary/DispersionTest.glb

The idea is to use the dispersion value to compute three separate IOR values for the red, green, and blue channel. The transmission computation is then evaluated per color channel with the respective IOR. The implementation is based on the glTF sample viewer [GLSL](https://github.com/KhronosGroup/glTF-Sample-Viewer/blob/6bc1df9c334288fb0d91d2febfddf97ac5dfd045/source/Renderer/shaders/ibl.glsl#L79).

I'll update the docs when the implementation is approved.

### CODE REVIEW

File: examples/webgl_loader_gltf_dispersion.html [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title>three.js webgl - GLTFloader + Punctual Lights</title>
```
Comment (donmccurdy) [inline examples/webgl_loader_gltf_dispersion.html:4]:
```suggestion
		<title>three.js webgl - GLTFloader + Dispersion</title>
```

File: examples/models/gltf/DispersionTest.glb
Comment (donmccurdy) [inline examples/models/gltf/DispersionTest.glb:1]: Perhaps an optimized version of the file? 

- original: 2.2 MB
- merge identical vertices: 1.4 MB
- merge identical vertices + meshopt: 0.4 MB

[Archive.zip](https://github.com/mrdoob/three.js/files/14843562/Archive.zip)

Comment (Mugen87) [reply examples/models/gltf/DispersionTest.glb:1]: I've used the `merge identical vertices` option in order to avoid a dependency to `MeshoptDecoder`.

**Comment by Mugen87** [comment]
@mrdoob The CI always fails with my version of the E2E screenshot for the new example `webgl_loader_gltf_dispersion`. Do you mind generating the screenshot on your system?

**Comment by Mugen87** [comment]
Never mind! I could fix the issue by generating the screenshot a different device, see #28062.

**Comment by donmccurdy** [comment]
The effect here looks excellent, thank you @Mugen87!!

**Pull request merged**